### PR TITLE
Change navigation bar color to white on api 27+

### LINKED ISSUE #10695: [Proposal] Change system navigation bar to match UI
### Expected behavior

I expected the system navigation bar (on the bottom of the screen) to adapt to the color of the UI, particularly the bottom navigation bar.

### Actual behavior

The system navigation bar is black by default, whereas our bottom navigation and UI in general is white or off-white. 

This tension is especially noticeable when you're using a device with rounded display corners where the navigation bar doesn't extend to account for the curvature – for example, when Gesture Navigation is turned on on a Pixel 4. Example:

image

Note: This isn't as much a problem in "3-button Navigation" mode because the height of that variation extends beyond the curvature of the corners. But I'm guessing (?) if we apply a style to one variation, it applies to others as well.

### Proposed Solution

Simply changing the system navigation bar so it provides a black indicator on a white background (specifically in light theme – dark theme would need a white indicator on system dark background) would basically solve this. Here's what it would look like in various places in the app (current version on left, proposed on right).

My site:

image

Editor:

image

Stats:

image

Browser:

image

##### Tested on [device], Android [version], WPAndroid [version]

Issue might be universally relevant, but currently testing on: Pixel 4, Android 10, WPAndroid 13.4

### PR DESCRIPTION
**Pull request by malinajirka**
Fixes #10695 

Changes color of navigation bar to white on Android 8.1+.

The first two columns are API  < 27 (without change), the last two columns are API >= 27:
Untitled
@iamthomasbishop I decided to keep the color white on all the screens for the sake of simplicity - it seems the only screens with grey-only background are the settings screens which aren't used that often anyway. 
We can enable drawing content below the navigation bar so scrolling content doesn't get cut off as part of another ticket. I tried to quickly implement it and I encountered several issues. So I think this change would need to be planned as part of a sprint.

To test:
1. Test the navigation bar color on a device with API < 27 and API >=27.

PR submission checklist:

- [x] I have considered adding unit tests where possible.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/res/values-v27/styles.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="WordPress" parent="Base.Wordpress">
+        <item name="android:navigationBarColor">@color/white</item>
```
Comment (theck13) [inline WordPress/src/main/res/values-v27/styles.xml]:
```suggestion
        <item name="android:navigationBarColor">@android:color/white</item>
```

Can we change this to `@android:color/white`? There is no resource white in `colors.xml` so using `@color/white` would be pulling from another source or library.

**Comment by iamthomasbishop** [comment]
@malinajirka Just played with a test build, and this already feels _much_ better. I'd be happy to ship as-is, but there might be refinements to be made with regards to the semi-transparency. 

One tiny thing I'm wondering if we should try is maybe putting a dim separator on top of the on the `3-button` navigation style. The short gesture navigation bar style works beautifully as you have here but the taller one might benefit from a separator. // cc @SylvesterWilmott @mattmiklic any thoughts/preferences?

**Comment by jd-alexander** [comment]
Tested on Pixel 4 XL device and it's looking good. Saw the same grey background on the Achievements page as shown below.
Screenshot_20191029-190159

 so yes, in the future would probably be good to make it all consistent but for now this change suits the light theme better.

**Comment by malinajirka** [comment]
> I'd be happy to ship as-is, but there might be refinements to be made with regards to the semi-transparency.

As I mentioned in the description, if we decide to make this change, I'd rather plan it as it won't be a matter of a couple of hours. We'll need to define which screens need to behave this way and we'll need to make sure actionable items don't appear bellow the bar. 

>One tiny thing I'm wondering if we should try is maybe putting a dim separator on top of the on the 3-button navigation style. The short gesture navigation bar style works beautifully as you have here but the taller one might benefit from a separator. 

Unless I'm missing a simply solution, I believe we'd need to modify layouts of all the screens in the app. So it might not be worth it.

@khaykov If it makes more sense for you to implement this in the dark-theme branch, please feel free to close this PR or change the target branch.

File: WordPress/src/main/res/values-v27/styles.xml (hunk 1 above)
Comment (malinajirka) [reply WordPress/src/main/res/values-v27/styles.xml]: 👍 Fixed in d11e529a33c8401d4a1f955723013d2a460e3e38

**Comment by SylvesterWilmott** [comment]
>One tiny thing I'm wondering if we should try is maybe putting a dim separator on top of the on the 3-button navigation style. The short gesture navigation bar style works beautifully as you have here but the taller one might benefit from a separator.

I don't see this being an issue, to be honest. However, Google does do this in its own apps by putting a line on the system navigation bar itself.

**Comment by iamthomasbishop** [comment]
> As I mentioned in the description, if we decide to make this change, I'd rather plan it as it won't be a matter of a couple of hours. We'll need to define which screens need to behave this way and we'll need to make sure actionable items don't appear bellow the bar.

@malinajirka Of course, I just mean the above solves the bulk of the issue so if/when we're ready we can move forward.

> FWIW, I've been considering moving to a white background (instead of gray) in light mode as part of the work I'm doing with @khaykov, so there may not be a need for a separator if we end up doing that.

@mattmiklic Ah, that's right. With that in mind, this approach should blend in nicely. 👍 

> However, Google does do this in its own apps by putting a line on the system navigation bar itself.

@SylvesterWilmott I wondered this, and when I checked out a handful of Google's apps, I found there isn't consistency – about half of them do have a separator and half don't, so I didn't know what would be preferred. I think it's fine without the separator as well, so we can roll with this.

**Comment by malinajirka** [comment]
@theck13 @jd-alexander I believe all the discussion are settled => it's ready for review/merge.
- we won't use separator for the navigation bar
- making the navigation bar semi transparent will be planned as regular task
- this change shouldn't collide with what Klym is working on

Thank you all ;)!

**Pull request merged**